drivers/power_domain/Kconfig

```diff
@@ -17,4 +17,8 @@ config POWER_DOMAIN_GPIO
 	depends on GPIO
 	depends on TIMEOUT_64BIT
 
+config POWER_DOMAIN_PSL
+	bool "Power Switch Logic (PSL) controlled power domain"
+	depends on GPIO
```

Comment:
this seems to be NPCX specific, so it is lacking a dependency.

Reply:
Done.